__tests__/server/plugins/reactHtml/createRequestHtmlFragment.spec.jsx

```diff
@@ -268,6 +274,21 @@ describe('createRequestHtmlFragment', () => {
     expect(renderForStaticMarkupSpy).not.toHaveBeenCalled();
   });
 
+  it('should throw an error if redirect location is not an allowed redirect url', async () => {
+    expect.assertions(3);
```

Comment:
not required with async await tests. expect.assertions is only necessary when an expect is potentially unreachable. an async function to `it` will fail the test if it does not resolve. 
```suggestion
```

Reply:
Removed in eb2c2281679bf7f5c23c4d2b1c8bfd0f879fd147